src/reducer/filterReducerFunc.js

```diff
@@ -0,0 +1,32 @@
+const defaultState = {
+    sortHighToLow: false,
+    sortLowToHigh: false,
+    rating: "4",
+    category: [],
+  };
+function filterReducerFunc (state,action) {
+    switch(action.type){
+        case "SORT":
+            return action.payload === "HIGH_TO_LOW"
+            ? { ...state, sortByHighToLow: true, sortByLowToHigh: false }
+            : { ...state, sortByHighToLow: false, sortByLowToHigh: true };
+        case "CATEGORY":
+            const { category } = state;
+            console.log(category)
```

Comment:
Remove console logs if not needed.

Reply:
Thank you for pointing it out. Just forget to remove these console logs